app.go

```diff
@@ -450,6 +450,9 @@ func (a *Application) InitChain(req abci.RequestInitChain) abci.ResponseInitChai
 			panic(err)
 		}
 	}
+	// load on-chain config
+	a.config = loadOnChainConfig(state)
```

Comment:
Don't need to load it here, it gets lazy loaded in `StoreState.Config()`

Reply:
Done